envs/conda-env-rapids.yml

```diff
@@ -0,0 +1,21 @@
+channels:
+  - rapidsai
+  - conda-forge
+  - nvidia
+  - nodefaults
+dependencies:
+  - python=3.10
```

Comment:
Matching comment on python versions

Reply:
RAPIDS env requires pinned version to avoid installation issues. Anyway, it's not used in CI currently.